pkg/addons/data/ImageStream.image.openshift.io/openshift-sdn/node.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: image.openshift.io/v1
```

Comment:
We need to figure out whether we are going to use image stream for infra updates...

Reply:
Open an issue to figure out whether we are going to use these image streams

Reply:
https://github.com/openshift/openshift-azure/issues/710